docs/website/docs/building-from-source/getting-started.md

```diff
@@ -396,6 +396,43 @@ cmake --build ../iree-build/
 
 ### Using the Python bindings
 
+There are two available methods for installing the Python bindings, either
+through creating an editable wheel or through extending `PYTHONPATH`.
+
+#### Installing the bindings as editable wheels
```

Comment:
Docs nit: how about prefixing these two options with "Option A:" and "Option B:"?

When scrolling through a list of instructions, it is easy to copy/paste both sets of instructions, when this is really a branch that developers should only choose one option from at a time.

![image](https://github.com/user-attachments/assets/a9732b69-c106-4d02-8063-c2ceb6c96a1a)

Reply:
Good point, done